src/features/home/pages/Home.test.tsx

```diff
@@ -16,30 +18,112 @@ describe('Home component', () => {
     navigate: jest.fn(),
   } as any // eslint-disable-line @typescript-eslint/no-explicit-any
 
-  it('should have a welcome message', async () => {
-    const { getByText } = render(<Home navigation={navigation} />)
+  afterAll(() => jest.resetAllMocks())
 
-    const welcomeText = await waitFor(() => getByText('Bienvenue à Pass Culture'))
-    expect(welcomeText.props.children).toBe('Bienvenue à Pass Culture')
-  })
+  describe('With current user loaded', () => {
+    beforeEach(() => {
+      jest.spyOn(api, 'useCurrentUser').mockImplementation(
```

Comment:
Si on peut gagner em lisibilité, ne pas hésiter : 
```suggestion
      jest.spyOn(api, 'useCurrentUser').mockReturnValue(
```

Reply:
Vu ensemble, c'est ok.